.eslintrc.json

```diff
@@ -19,10 +19,23 @@
   "settings": {
     "version": "detect"
   },
-  "plugins": ["react"],
+  "plugins": [
+    "react",
+    "import-newlines"
+  ],
   "rules": {
     "semi": ["error", "always"],
     "quotes": ["warn", "single"],
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "object-curly-spacing": ["warn", "always"],
+    "import-newlines/enforce": [
+      "warn",
+      {
+        "items": 1,
+        "max-len": 100,
+        "semi": false
+      }
+    ],
+    "indent": ["warn", 4]
```

Comment:
need we this config when our prettier's new line config length is set to 90?

Reply:
I did not understand what you wrote.

Reply:
@TigranYengiba04 jan, Igor says that we already have a prettier config for max length (`printWidth`) and it's set to '100' (actually). Please look into `.prettierrc` file.
Also rules about `indent` and `semicolon` that you specified in eslint file. These rules are already set in our .prettierrc file as well. It's not visible in it, because these rules are set by default [from prettier](https://prettier.io/docs/en/options.html).

I think we shouldn't show any warning or error in case of indentation, semicolon or max length violations by developers. That's why I left these rules in prettier and didn't write it in eslint. But if you think we should, then let's discuss.

Reply:
I think we should keep our code style config in "eslint" or "prettierrc". I think it's better to have single source for code styling.

Reply:
Do you mean we should use either Eslint or Prettier? I agree we should have one config file, but this is a different case. Both Eslint and Prettier do different jobs. Eslint's responsibility is to show some warnings or errors if there are any violations by developers, that's all. Prettier's responsibility is to format and give code style to the written code, so everyone's code has the same styling after all. 
For example, the configs I've specified in configuration files - without eslint we would have many unused imports or variables, without prettier we would have unformatted code.
See the file change of the comment above, in which the Prettier hadn't worked.
In my opinion, they work great together. It's especially visible in pre-commit hook.

Reply:
Agree with @aninaslyan they have different responsibilities.

Reply:
done